tests/unit/commands/__init__.py

```diff
@@ -124,3 +133,69 @@ def setUp(self):
         self.fake_store = fixture_setup.FakeStore()
         self.useFixture(self.fake_store)
         self.client = storeapi.StoreClient()
+
+
+class FakeStoreCommandsBaseTestCase(CommandBaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.fake_store_login = fixtures.MockPatchObject(storeapi.StoreClient, "login")
+        self.useFixture(self.fake_store_login)
+
+        self.fake_store_register = fixtures.MockPatchObject(
+            storeapi._sca_client.SCAClient, "register"
+        )
+        self.useFixture(self.fake_store_register)
+
+        self.fake_store_account_info = fixtures.MockPatchObject(
+            storeapi._sca_client.SCAClient,
+            "get_account_information",
+            return_value={
+                "account_id": "abcd",
+                "account_keys": list(),
+                "snaps": {
+                    "16": {
+                        "snap-test": {
+                            "snap-id": "snap-test-snap-id",
+                            "status": "Approved",
+                            "private": False,
+                            "since": "2016-12-12T01:01Z",
+                            "price": "0",
+                        }
+                    }
+                },
+            },
+        )
+        self.useFixture(self.fake_store_account_info)
+
+        self.fake_store_status = fixtures.MockPatchObject(
+            storeapi._sca_client.SCAClient, "snap_status", return_value=dict()
+        )
+        self.useFixture(self.fake_store_status)
+
+        self.fake_store_revisions = fixtures.MockPatchObject(
+            storeapi._sca_client.SCAClient, "snap_revisions", return_value=dict()
+        )
+        self.useFixture(self.fake_store_revisions)
+
+        self.fake_store_release = fixtures.MockPatchObject(
+            storeapi.StoreClient, "release"
+        )
+        self.useFixture(self.fake_store_release)
+
+        self.fake_store_register_key = fixtures.MockPatchObject(
+            storeapi._sca_client.SCAClient, "register_key"
+        )
+        self.useFixture(self.fake_store_register_key)
+
+        # Mock the snap command
+        self.fake_check_output = fixtures.MockPatch(
```

Comment:
Just the `snap` command?

Reply:
Yes, the side_effect mocks the snap command, let's unsquashfs pass and fails on every other call